src/common/utils.ts

```diff
@@ -2,7 +2,8 @@ import * as _ from 'lodash'
 import BigNumber from 'bignumber.js'
 import {deriveKeypair} from 'ripple-keypairs'
 import {Amount, RippledAmount} from './types/objects'
-import {ValidationError} from './errors'
+import {RippledError, ValidationError} from './errors'
+import { isValidXAddress, xAddressToClassicAddress } from 'ripple-address-codec'
```

Comment:
We should try to make the linter/prettier make the spacing consistent here

Reply:
That's a separate task,  though, and I believe we have updating the linter triaged and in our backlog.